Sources/MuxPlayerSwift/PublicAPI/Options/PlaybackOptions.swift

```diff
@@ -82,6 +82,9 @@ public struct InstantClipping: Equatable {
     var assetStartTimeInSeconds: Double
     var assetEndTimeInSeconds: Double
 
+    var programStartTimeEpochInSeconds: Double
+    var programEndTimeEpochInSeconds: Double
+
     var noInstantClipping: Bool {
         return self.assetStartTimeInSeconds.isNaN && self.assetEndTimeInSeconds.isNaN
```

Comment:
nitpick(blocking): Update computed property to account for pdt props

Reply:
good catch [40b6422](https://github.com/muxinc/mux-player-swift/pull/63/commits/40b6422a71f50cea890017556c8cd60ad04fa6ca)